docs/installation.rst

```diff
@@ -0,0 +1,51 @@
+.. highlight:: shell
+
+============
+Installation
+============
+
+
+Stable release
+--------------
+
+To install grafana-inviter, run this command in your terminal:
+
+.. code-block:: console
+
+    $ pip install grafana_inviter
```

Comment:
This is only Python3 correct? We should probably mention that it must be a pip instance associated with Python3, as by default on some systems it is still pip2.

Reply:
Should be not needed as the README shows via the badges all supported versions and if you try to install it with python27 it will fail